projects/browser-extension/src/heads-up/index.js

```diff
@@ -1,9 +1,9 @@
 import Vue from 'vue';
-import App from './App.vue';
 import * as actions from './lib/message-actions';
 import router from './router';
 import VTooltip from 'v-tooltip';
 import { createStore } from './store';
+import AppShell from './app-shell.vue';
```

Comment:
The name `App` didn't fit the projects' own naming convention.
I couldn't move this file into the components folder however, since it gave some very mysterious import errors.